src/crab/ebpf_transformer.cpp

```
@@ -132,7 +132,20 @@ class ebpf_transformer final {
                       const linear_expression_t& val_uvalue, const std::optional<reg_pack_t>& opt_val_reg);
 }; // end ebpf_domain_t
 
-void ebpf_domain_transform(ebpf_domain_t& inv, const Instruction& ins) { std::visit(ebpf_transformer{inv}, ins); }
+void ebpf_domain_transform(ebpf_domain_t& inv, const Instruction& ins) {
```

cppcheck

[style] The function 'ebpf_domain_transform' is never used.

(unusedFunction)